### PR TITLE
use apollo built-in health check

### PR DESCRIPTION
**Pull request by redreceipt**

### CODE REVIEW

File: apollos-church-api/src/server.js [hunk 1]
```diff
@@ -74,22 +71,6 @@ const apolloServer = new ApolloServer({
 
 const app = express();
 
-// health check
-app.get('/health', cors(), (req, res) => {
-  res.send('ok');
-});
-
-// apollos version
-app.get('/version', cors(), (req, res) => {
```
Comment (vinnyjth) [inline apollos-church-api/src/server.js:83]: Do you remember why we had this version check in here?

**Review by vinnyjth — APPROVED** [review]

**Comment by redreceipt** [comment]
Yea I put I there when we were gonna do an admin panel with a status check of all the apps. But we can just look at the Apollos.JSON file if we ever get around to doing something like that.

**Comment by redreceipt** [comment]
@vinnyjth I just removed sqlite completely in this PR, that's ok right?

**Comment by vinnyjth** [comment]
> @vinnyjth I just removed sqlite completely in this PR, that's ok right?

Yep! that should be good, we've moved past supporting it as a default.

**Pull request merged**